Gemfile.lock

```diff
@@ -437,6 +437,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-23
+  arm64-darwin-24
```

Comment:
I don’t know what this line is doing, do you? I’m not opposed to adding it if it’s helping something, but if it’s just a stray line we should remove it.

Reply:
That was added automatically. I just assumed cause I'm on a mac m1. I can take it out ... maybe :) I'll look into git-ing it out.

Reply:
I removed the arm64-darwin-24. I think lol. Thanks for your patience.

Reply:
Running the test, it tries to stuff it back in there.

Reply:
Some quick googling suggests that you may be on a newer version of the Mac operating system and that’s why it’s added, but I didn’t dig into it much. It’s probably fine to leave it but it’s also fine to take it out.